PlayKitSmartSwitch/Classes/SmartSwitchPluginConfig.swift

```diff
@@ -59,8 +70,8 @@ import Foundation
     }
     
     @discardableResult
-    @nonobjc public func set(smartSwitchUrl: String) -> Self {
-        self.smartSwitchUrl = smartSwitchUrl
+    @nonobjc public func set(domainUrl: String) -> Self {
+        self.domainUrl = domainUrl
         return self
     }
```

Comment:
Are we assuming that if someone calls this function it will replace the domainUrl and they need to provide it with '{accountCode}/{application}/decision' at the end? Is this mandatory data? Do we need to append it if it is not in the url?